can_bus/quadruna/VC/VC_tx.json

```diff
@@ -13,7 +13,7 @@
         }
     },
     "LeftInverterCommand": {
-        "msg_id": 32,
+        "msg_id": 201,
```

Comment:
These message IDs are actually special, which is why they're not in the usual 200s range. The inverter needs to know the specific ID of its command message, which is currently programmed to be 32/82. 

This is also why we're so specific in the signal config, because we have to conform exactly to our inverter's spec.

Reply:
Fixed.